composer.json

```diff
@@ -18,6 +18,9 @@
     "require-dev": {
         "doctrine/cache": "~1.0"
     },
+    "suggest": {
+        "cache/adapter-common": "To make use of PSR-6 cache implementation via PsrCacheAdapter."
```

Comment:
It should be `psr/cache-implementation` to enable any PSR cache =)

https://packagist.org/providers/psr/cache-implementation